examples/offline_inference_vision_language.py

```diff
@@ -385,7 +385,12 @@ def run_idefics3(question: str, modality: str):
     llm = LLM(model=model_name,
               max_model_len=8192,
               max_num_seqs=2,
-              enforce_eager=True)
+              enforce_eager=True,
+              mm_processor_kwargs={
```

Comment:
Should we add comments to explain why we need to add this parameter?

Reply:
Sure.